New log file opened at Sun May 10 13:44:15 CEST 2015 by 5060@Julia
Taking next connection to: 56532
New client attempting to connect on port: 56533-56532
  Address = /10.0.0.3:56532
This client on port 56533 has uuid 68012932-fb28-4485-8c33-1927a77aad45 configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-68012932-fb28-4485-8c33-1927a77aad45
Taking next connection to: 56532
New client attempting to connect on port: 56534-56532
  Address = /10.0.0.3:56532
Handshake exception on socket: 56534
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 56532
New client attempting to connect on port: 56535-56532
  Address = /10.0.0.3:56532
This client on port 56535 has uuid 9b7ef34f-31c8-4ca2-8d62-0fae3ae4f040 configName activator humanReadableName Activator protocolVersion 1 featureTags Vector()
Connected Clients: activator-68012932-fb28-4485-8c33-1927a77aad45, activator-9b7ef34f-31c8-4ca2-8d62-0fae3ae4f040
Taking next connection to: 56532
New client attempting to connect on port: 56539-56532
  Address = /10.0.0.3:56532
Handshake exception on socket: 56539
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 56532
New client attempting to connect on port: 56540-56532
  Address = /10.0.0.3:56532
This client on port 56540 has uuid 21a826d1-81b1-476b-ae21-77ca2bd5540c configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-68012932-fb28-4485-8c33-1927a77aad45, activator-9b7ef34f-31c8-4ca2-8d62-0fae3ae4f040, play-fork-run-21a826d1-81b1-476b-ae21-77ca2bd5540c
Taking next connection to: 56532
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 56532
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 56532
